### ISSUE TITLE
Search engine in sidebar instead of popup [docs].

### ISSUE THREAD

**Issue by lukaszpolowczyk** [sveltejs/kit#3948]
### Describe the problem

Please see the search engine for this project:
https://mermaid-js.github.io/mermaid/#/entityRelationshipDiagram?id=syntax

She is better under ABSOLUTELY EVERYTHING. :DD

Please, I really wish SvelteKit had a search engine like this. 🙏

![obraz](https://user-images.githubusercontent.com/16800535/154230947-5d7fb76e-4fa9-41a1-a5e7-264a77abebf2.png)

### Describe the proposed solution

* search engine in side panel
* search results in the side panel
* search engine does not disappear when I preview the searched page

### Alternatives considered

You can't think of anything better. :P

### Importance

would make my life easier

**Comment by Bishwas-py** [comment]
Yes, can be done. But where the work/code should be done? Any repo?

**Comment by benmccann** [comment]
The code for the site lives here: https://github.com/sveltejs/kit/tree/master/sites/kit.svelte.dev

However, it would be good to get agreement on the implementation before spending time coding anything. When I visit https://mermaid.js.org/, their search looks almost identical to ours, so I don't understand what is being proposed. I would guess they have redesigned their search since this issue was originally posted, which now makes this issue impossible to understand.

**Comment by lukaszpolowczyk** [comment]
@benmccann I took a screen shot, it is visible in the issue.
In the screen shot, "uml" is typed in the search box, the search box is in the right side panel, and the search results are also in the left side panel.

In fact, mermaid has changed its search engine. In my opinion, now mermaid has a worse search engine than before.

The search engine in the left side panel allows you to view search results more easily, because they are not obscured by the search engine popup in any way, and it is easy to view search results because they do not disappear, even when you enter a searched sub-page.

I found this to be a much more convenient way to search for something.
It's a shame that the mermaid people changed their minds, but that's their problem. In the SvelteKit documentation it can be as you decide, but I would prefer a search engine in the right side panel.
